third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -507,7 +533,8 @@ PjRtComputationClient::ExecuteReplicated(
     data_handles.push_back(datas);
   }
 
-  TF_VLOG(1) << "Returning " << data_handles.size() << " sets of results";
+  TF_VLOG(1) << "Returning " << data_handles.size() << " sets of results "
```

Comment:
This will be quite verbose, no? Maybe make it a 2 or 3? IMO the 1 should only contains the critial messages like "I init", "I execute". Details debug information should be in higher levels.

Reply:
Ack.

Reply:
Actually, this will be not that lengthy/verbose, and it's on par with unsharded computation logging: https://github.com/pytorch/xla/blob/a6d186f7c6f27b94d6357b4d7ef50683b10612d7/third_party/xla_client/pjrt_computation_client.cc#L446